### PR TITLE
added first two energizers

### PR DESCRIPTION
**Pull request by jonleung**

### CODE REVIEW

File: cohort_4/playbook/ACTIVITIES.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 **Table of Contents**
 
 - [Hacker arrival](#hacker-arrival)
+- [Energizers](#energizers)
+  - [Where The West Wind Blows](#where-the-west-wind-blows)
+  - [I Love My Neighbor Who ___](#i-love-my-neighbor-who-___)
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: Can you indent this 4 spaces like the rest of the TOC? I know it doesn't follow our markdown style guide, but making this 4 spaces would be consistent with what we currently have.

File: cohort_4/playbook/ACTIVITIES.md [hunk 2]
```diff
@@ -50,6 +53,50 @@
...
+
+## Human Pictionary
+
+```
+Time: 10 min
+Materials: None
+Team Sizes: 
+  - when 4-8 people, group into teams of size 2
+  - when 9-12 people, group into teams of size 3
+  - when 13+ people, group into teams of size 4 or 5
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md:71]: Can you reformat these bullet points to follow our markdown style guide?

```
...
Materials: None
Team Sizes:

- when 4-8 people, group into...
```

File: cohort_4/playbook/ACTIVITIES.md [hunk 3]
```diff
@@ -50,6 +53,50 @@
...
+# Energizers
+
+## Where The West Wind Blows
+
+## Human Pictionary
+
+```
+Time: 10 min
+Materials: None
+Team Sizes:
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: Should "Sizes" be capitalized?

File: cohort_4/playbook/ACTIVITIES.md [hunk 4]
```diff
@@ -50,6 +53,50 @@
...
+```
+Time: 5 to 15 minutes
+Materials: None
+Number: Best for 10-35
+```
+
+**Facilitation:**
+
+- Get everyone into a circle
+- Facilitator starts "in the middle" by filling in “My name is ______. I love my neighbor who _____” with anything that applies to him/her such as “I love my neighbor who dances.” or “I love my neighbor who loves sushi”.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: Can you reflow this text to have an 80 character line limit, per our markdown style guide?

File: cohort_4/playbook/SCHEDULE.md [hunk 5]
```diff
@@ -64,12 +64,12 @@ will be followed by project-time that day.
 | 8:30 AM  | Standups                                                     |
 | 9:20 AM  | [Hacker arrival](ACTIVITIES.md#hacker-arrival)               |
 | 9:30 AM  | Galvanize tour                                               |
-| 9:40 AM  | Energizer                                                    |
+| 9:40 AM  | [Energizer 1](ACTIVITIES.md#where-the-west-wind-blows) & [Energizer 2](#i-love-my-neighbor-who-___)                                 |
```
Comment (zachlatta) [inline cohort_4/playbook/SCHEDULE.md]: The second link is broken. Can you also reformat the markdown table to work with this increased width?

File: cohort_4/playbook/SCHEDULE.md [hunk 6]
```diff
@@ -64,12 +64,12 @@ will be followed by project-time that day.
...
 | 10:00 AM | [Breakfast](ACTIVITIES.md#breakfast)                         |
 | 10:30 AM | Introduction talk                                            |
 | 10:40 AM | [Portfolio workshop](ACTIVITIES.md#portfolio)                |
 | 12:30 PM | [Lunch](ACTIVITIES.md#lunch)                                 |
-| 1:30 PM  | Energizer                                                    |
+| 1:30 PM  | [Energizer 3](ACTIVITIES#evolution-rock-paper-scissors)      |
```
Comment (zachlatta) [inline cohort_4/playbook/SCHEDULE.md]: This link is broken. Perhaps you meant `ACTIVITIES.md`?

File: cohort_4/playbook/ACTIVITIES.md [hunk 7]
```diff
@@ -50,6 +54,63 @@
...
+```
+Time: 5 to 15 minutes
+Materials: None
+Number: Best for 10-30
+Source: Andrea
+```
+
+**Facilitation**
+
+Everyone starts off in the “egg” stage and has to act like eggs like being in a squatting position. Every “egg” will find another “egg” to play rock paper scissors with. Play rock paper scissors 3 times (one round) to see if either wins 2 out of 3. If not, play another round etc. until someone wins a round 2 out of 3. The person that wins will then evolve into the next stage “monkey” and has to find another “monkey” to play rock paper scissors with. The “losing” person will remain an egg and find other eggs to play rock paper scissors with. The stages are egg to monkey to human then to divine being. In each stage, the “losing” person will de-evolve into the previous stage or if an egg will remain an egg. Once a person becomes a divine being he/she leaves the game and watches. Continue until you have a certain number of divine beings agreed upon at the beginning of the game.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: Can you reflow this text to 80 character line limit?

File: cohort_4/playbook/ACTIVITIES.md [hunk 8]
```diff
@@ -50,6 +54,63 @@
...
+- Announce that there is no talking in this activity
+- Facilitator(s) will call out a random prompt (e.g. iPhone, giraffe, surfboard) and start the 30 second timer. 
+- Each team tries to represent that the prompt with their bodies in that time. 
+- At the end of the 30 seconds, everyone will freeze as the facilitator(s) goes around and judges the teams and makes funny and comments. 
+  - "This group is trying really hard not to fall. It really does look like th leaning tower of Pizza. Bonus points on being very angled."
+- After observing all the groups, the facilitator announces the winner.
+- The facilitator tells the winning team gets to pick, announce, and judge the next round.
+- You can repeat this for the duration you want this activity.
+
+## I Love My Neighbor Who ___
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: What is the ___ for?

Perhaps `I Love My Neighbor Who...` might be a bit more clear?

File: cohort_4/playbook/ACTIVITIES.md (hunk 1 above)
Comment (jonleung) [inline cohort_4/playbook/ACTIVITIES.md]: Cool

File: cohort_4/playbook/ACTIVITIES.md (hunk 2 above)
Comment (jonleung) [inline cohort_4/playbook/ACTIVITIES.md:71]: yup

File: cohort_4/playbook/ACTIVITIES.md (hunk 3 above)
Comment (jonleung) [inline cohort_4/playbook/ACTIVITIES.md]: yes?

File: cohort_4/playbook/SCHEDULE.md (hunk 5 above)
Comment (jonleung) [inline cohort_4/playbook/SCHEDULE.md]: @zachlatta 
- [x] I can't figure out how to make the link work for [Energizer 2](#i-love-my-neighbor-who-___) and

File: cohort_4/playbook/SCHEDULE.md (hunk 6 above)
Comment (jonleung) [inline cohort_4/playbook/SCHEDULE.md]: Thanks!

File: cohort_4/playbook/ACTIVITIES.md (hunk 7 above)
Comment (jonleung) [inline cohort_4/playbook/ACTIVITIES.md]: yup

**Comment by jonleung** [comment]
@zachlatta can you help me take a look at this again? Thanks!

**Comment by maxwofford** [comment]
@jonleung Would you please address the merge conflicts?

**Comment by maxwofford** [comment]
@jonleung I'll review this and merge it.

EDIT: nvm

**Pull request closed without merging**